compiler/src/iree/compiler/Codegen/Common/GPU/GPUHeuristics.h

```diff
@@ -23,15 +23,16 @@ struct GPUMatmulShapeType {
 
 /// Struct containing seed tile sizes for GPU MMA heuristics deduction logic.
 struct GPUMMAHeuristicSeeds {
-  // The default number of subgroups to use per workgroup
-  int64_t numSubgroupsPerWorkgroup;
-  // The default number of tiles along M/N dimension to use per workgroup
-  int64_t numMNTilesPerSubgroup;
-  // The default number of tiles along K dimension to use per subgroup
-  int64_t numKTilesPerSubgroup;
+  // The best number of subgroups to use per workgroup
+  int64_t bestSubgroupCountPerWorkgroup;
+  // The best number of total tiles along M*N dimensions per subgroup
+  int64_t bestMNTileCountPerSubgroup;
+  // The best number of tiles along K dimension per subgroup
+  int64_t bestKTileCountPerSubgroup;
 };
 
 struct GPUMMASchedule {
+  uint64_t index;     // Index of the chosen intrinsic
```

Comment:
Actually that's what I want to do (next). Need to define a layout for cooperative matrix so we can send in an array of `MmaAttr` directly. But chosen this way for now as just an intermediate step to avoid converting the whole world. :)

Reply:
splitting things up makes sense, thanks for clarifying.